app/frontend/src/components/base/BaseSecure.vue

```diff
@@ -32,15 +32,15 @@ export default {
       'createLoginUrl',
       'isAdmin',
       'keycloakReady'
-    ]),
-    authorized() {
-      return this.admin ? this.isAdmin : true;
-    }
+    ])
   },
   methods: {
+    authorized() {
+      return this.admin ? this.isAdmin : true;
```

Comment:
if authorized is only the in the case where admin is true, then shouldn't this be called admin()?  are we saying the the deault use of this component is for admins only? but other components could change authorized() to check if some other value(s) were true?

Reply:
I left this as authorized() because this function is actually intended to check for resource roles in the long run. At this time, Getok is tied to realm roles instead, so looking for the correct roles has to look at the realm level. The overall intent is that this authorization function would look more along the lines of what is in silvipc (refer to the BaseSecure implementation version) which is why the function is still named authorized. However in this case, the implementation retains the simpler isAdmin check to retain current functionality.

We're going to have to quickly decide on getok if we want to swap from realm level to resource level roles, or if we want to leave it as is before it gets developed too much further.